plugins/dead_code_elimination/dead_code_elimination_main.ml

```diff
@@ -24,8 +25,9 @@ let computed_def_use sub =
   def_use_collector#visit_sub sub (Var.Set.empty,Var.Set.empty)
 
 let sub_args sub =
-  Term.enum arg_t sub |>
+  Term.enum arg_t sub |> Seq.to_list |>
```

Comment:
there is no need to transform it to `list`. I was asking only to use a list instead of a memoized sequence for the list of subroutines. Not like use list everywhere.